app/assets/stylesheets/views/notifications.scss

```diff
@@ -0,0 +1,148 @@
+@import '../config/import';
+
+.notification {
+  --unseen-border-width: 0px;
+  --padding: var(--su-3);
+  padding: var(--padding);
+  padding-left: calc(var(--padding) - var(--unseen-border-width));
+
+  display: flex;
+  color: var(--base-80);
+  border-radius: 0;
+
+  @media (min-width: $breakpoint-s) {
+    --padding: var(--su-4);
+  }
+  @media (min-width: $breakpoint-m) {
+    --padding: var(--su-6);
+    border-radius: var(--radius);
+  }
+
+  &--middle {
+    align-items: center;
+  }
+
+  &--promoted {
+    text-align: center;
+    flex-direction: column;
+    align-items: center;
+    padding: var(--su-6);
+
+    @media (min-width: $breakpoint-m) {
+      padding: var(--su-8);
+    }
+  }
+
+  &__content {
+    padding-left: var(--su-4);
+    flex: 1 auto;
+    // Hack preventing content exceeding its container - it works for scenarios where we have
+    // `pre` tag having wide (overflowing) text. And the container itself is part of a layout
+    // built using CSS `grid`. Without this, the grid layout wouldn't stick to its defined
+    // widths...
+    min-width: 0;
+  }
+
+  &__mod-controls {
+    padding: var(--su-1);
+    display: flex;
+    align-items: center;
+    font-size: var(--fs-s);
+    position: absolute;
+    right: calc(var(--su-2) * -1);
+    top: calc(var(--su-3) * -1);
+    border-radius: 9999px;
+  }
+
+  &__actions {
+    border-top: 1px solid var(--base-10);
+    padding-top: var(--su-1);
+    display: flex;
+  }
+
+  &__preview {
+    padding: var(--su-1);
+
+    &__title {
+      font-size: var(--fs-xl);
+      line-height: var(--lh-tight);
+      margin-bottom: var(--su-2);
+
+      @media (min-width: $breakpoint-s) {
+        font-size: var(--fs-2xl);
+      }
+
+      @media (min-width: $breakpoint-l) {
+        font-size: var(--fs-3xl);
```

Comment:
When I first saw the post notifications, something felt "off" to me with the typography, and I think it's that we have some lack of uniformity across type sizes here. I'd recommend myself this more uniform to the way we show posts in feed (not the featured post, but just the standard ones).

```suggestion
        font-size: var(--fs-2xl);
```

![Screen Shot 2020-10-27 at 11 17 57 AM](https://user-images.githubusercontent.com/816402/97345124-ee511280-1846-11eb-9977-ece0cb3a2758.png)
![Screen Shot 2020-10-27 at 11 18 05 AM](https://user-images.githubusercontent.com/816402/97345126-f0b36c80-1846-11eb-93ea-7f1d148e8122.png)
![Screen Shot 2020-10-27 at 11 18 13 AM](https://user-images.githubusercontent.com/816402/97345130-f14c0300-1846-11eb-8ef4-a475f50d61d8.png)

Reply:
I can decrease it, no problems. Although, keep in mind that notification cards are a bit wider than the feed cards so bumping up the font size is somehow justified.